### ISSUE TITLE
Java 17 Support

### ISSUE THREAD

**Issue by larrywalters** [seujorgenochurras/image-to-ascii#14]
Your project looks very useful, but when trying to use it in a Java 17 project, the following error is displayed:
```
  bad class file: ~/.m2/repository/io/github/seujorgenochurras/image-to-ascii/0.0.4/image-to-ascii-0.0.4.jar(/io/github/seujorgenochurras/DefaultAsciifier.class)
    class file has wrong version 64.0, should be 61.0
    Please remove or make sure it appears in the correct subdirectory of the classpath.
```
So the project is built with Java 21, and indeed, when I use that version, it compiles successfully.

Since Java 17 is still widely used and considered LTS, it would be useful to have the project, or at least a specific release branch, built using Java 17. Some developers can have restrictions which prevent them from using Java 21.

If I'm misunderstanding and there is a way to work around this issue, please let me know.

Thanks.

**Comment by seujorgenochurras** [comment]
I'm sorry it took this long, but I'm going to downgrade the library to jdk 17

**Comment by seujorgenochurras** [comment]
I mean jdk 21 is the latest LTS but  java community takes so fucking long to update that it is definitely better to just downgrade it

**Comment by seujorgenochurras** [comment]
Fixed on latest release #15
